api/server/routes/rest/lib/validation.js

```diff
@@ -2,7 +2,7 @@ const httpErrors = require('http-errors');
 const {
 	validators,
 	TreecreeperUserError,
-} = require('../../../../../packages/schema-sdk');
+} = require('@financial-times/tc-schema-sdk'); // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
I'm not sure why eslint tells error but need disabling otherwise linting will fail